content/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github.md

```diff
@@ -59,6 +59,11 @@ If a site administrator disables the merge conflict editor for pull requests bet
    Click **Create branch and update my pull request** or **I understand, continue updating _BRANCH_**. The button text corresponds to the action you are performing.
 1. To merge your pull request, click **Merge pull request**. For more information about other pull request merge options, see "[Merging a pull request](/articles/merging-a-pull-request/)."
 
+ If the conflict is in binary data such as an image or sound file, this method of resolving conflicts cannot be used. 
+ In order to resolve conflicts involving binary files, immidiately make a back up, delete and re-clone the project, then re-introduce non-conflicting files.
```

Comment:
I'm not sure that advising the user to add non-conflicting files is particularly useful in this article. This also assumes the user is working from a clone, which may not be the case.